app/src/main/java/io/beerdeddevs/heartbeards/feature/timeline/TimelineAdapter.kt

```diff
@@ -15,30 +15,29 @@ import com.jakewharton.picasso.OkHttp3Downloader
 import com.squareup.picasso.Picasso
 import io.beerdeddevs.heartbeards.R
 
+class TimelineAdapter(context: Context, options: FirebaseRecyclerOptions<TimelineItem>)
+    : FirebaseRecyclerAdapter<TimelineItem, TimelineAdapter.ViewHolder>(options) {
 
-class TimelineAdapter(context: Context, options: FirebaseRecyclerOptions<TimelineItem>) :
-        FirebaseRecyclerAdapter<TimelineItem, TimelineAdapter.ViewHolder>(options) {
-
-    private val picasso = Picasso.Builder(context).downloader(OkHttp3Downloader(context)).build()
-    private val layoutInflater = LayoutInflater.from(context)
     private val width: Int
     private val height: Int
+    private val layoutInflater = LayoutInflater.from(context)
+    private val picasso = Picasso.Builder(context)
+            .downloader(OkHttp3Downloader(context))
+            .build()
 
     init {
         val display = (context as Activity).windowManager.defaultDisplay
-        val size = Point()
-        display.getSize(size)
+        val size = Point().apply { display.getSize(this) }
```

Comment:
Shouldn't we use run here?

Reply:
Nope, `run` returns something else then object itself. 
any `getSize` changes the Point object, so eventually we modify the object we created in first place, which is a great candidate for `apply` :)